### DISCUSSION TITLE
Meeting Reschedule - Friday 11/10/2024

### DISCUSSION THREAD

**Discussion by Rayccheal** [clentafrica/hr-people-operations#10] [Polls]
Dear Team Leads, @clentafrica/wg-design @clentafrica/wg-disability-gender-equality @clentafrica/wg-education @clentafrica/wg-energy @clentafrica/wg-hr @clentafrica/wg-health @clentafrica/wg-strategy @clentafrica/wg-monitoring-and-evaluation @clentafrica/wg-gsoc @clentafrica/wg-fundraising @clentafrica/wg-sustainability-esg @clentafrica/wg-gsod @clentafrica/wg-web-dev @clentafrica/wg-programs @clentafrica/wg-research-and-data @clentafrica/wg-marketing-comms @clentafrica/wg-community-managers @clentafrica/wg-energy @yigakpoa @o

Thank you so much for your feedback!

From the responses we received in the previous discussion and the email sent out, we identified some challenges that may hinder our ability to meet effectively, including time zone differences and the unavailability of some team members during weekends and at certain times of the day.

With this in mind, I have created a poll that we hope will help us arrive at a suitable time for everyone.

Thank you, and I look forward to meeting with all of you on **Friday (if that is a convenient time)** to share our brilliant ideas for the betterment of the team!

Thank you for your understanding!

P.S. Please check with your time zone(s) [here](https://www.worldtimebuddy.com/?pl=1&lid=100,5,2332459,30,8&h=100&hf=1) before you vote.

Regards,
Racheal

**Comment by Maxewa** [comment]
What time on Friday

**Reply by yigakpoa** [reply]
Please vote using the poll above 👆

**Reply by Maxewa** [reply]
There is no poll link anywhere here

**Reply by yigakpoa** [reply]
![IMG_5509](https://github.com/user-attachments/assets/d36fbee5-e7a9-44d7-ac1c-b5d4ff915042)

@Maxewa Click on vote at the right side.

**Reply by yigakpoa** [reply]
Hi @Fathalrhman26 @kola42 @MadhalasaSJ @Jonnie-Dev @SDauphin15 @ALIPHATICHYD @aramide-dess @Ladique @EmmanetD @Okpeelizabeth1989 please can you vote using the polls here. 

The other discussion is now locked! 

Thank you

**Reply by Rayccheal** [reply]
Thank you so much @yigakpoa !!!!
Adding the time zone calculator is such a lifesaver!!!
Thank you so much!

Thank you @everyone for your votes as well. We hope to get more votes soon but if not, we will make do with whatever we have arrived at.

Thank you all so much, and I look forward to our meeting.

**Comment by Maxewa** [comment]
I voted already. I appreciate

**Comment by Obutorbeatrice** [comment]
Link closed?

**Reply by Rayccheal** [reply]
No. It's not. 

You will see the poll below the message body.

Please vote. Thank you!

**Comment by Rayccheal** [comment]
Hi! @mbomdaS , @uchiha-vivek , @emylajem , @JuliaKamila02 , @Abraham100000 ,

This is the place to vote for the time.

PS: The new date is Friday - 11/10/24. 

Let's just vote to decide the time that would be most convenient for us.

Thank you!

**Comment by JuliaKamila02** [comment]
Hi,

I have already voted.

Best,
Julia

**Reply by Rayccheal** [reply]
Thank you Julia, and I look forward to seeing you at the meeting!!

**Comment by Okpeelizabeth1989** [comment]
I have voted

**Comment by Rayccheal** [comment]
Dear Team Leads, @clentafrica/wg-design @clentafrica/wg-disability-gender-equality @clentafrica/wg-education @clentafrica/wg-fundraising @clentafrica/wg-web-dev @clentafrica/wg-energy @clentafrica/wg-partnerships @clentafrica/wg-monitoring-and-evaluation @clentafrica/wg-health @clentafrica/wg-sustainability-esg @clentafrica/wg-strategy @clentafrica/wg-gsoc @clentafrica/wg-gsod @clentafrica/wg-hr @Onodjayeke 

Thank you for your valuable feedback thus far. We are close to reaching a decision; however, we currently find ourselves in a 50/50 situation.

To help us finalize a meeting time, please take a moment to participate in the poll if you have not already done so. Your input is crucial for determining a specific time.

Thank you for your collaboration and support

Regards,
Racheal
